inst/extdata/energy/A15.roofPV_TechChange.csv

```diff
@@ -0,0 +1,5 @@
+# File: A15.roofPV_TechChange.csv,,,,,,,
+# Title: GCAM 3.0 rooftop PV techChange,,,,,,,
+# Units: Unitless,,,,,,,
+resource,subresource,2010,2025,2040,2055,2070,2085
```

Comment:
Add a `Description:` and/or `Source` field please, specifying where these numbers come from and what they mean. Same comment applies to following file too.